### ISSUE TITLE
과제 키워드 Wiki에 만들기

### ISSUE THREAD

**Issue by 42-mki** [42-mki/42seoul_study_machine_learning#1]
# 개발 환경
- Anaconda
- Tensorflow
- Jupyter notebook

# 머신러닝의 분류
- 지도 학습(supervised learning)과 비지도 학습(unsupervised learning)
- 강화 학습
- scikit-learn

# 딥 러닝
- 인공 신경망
- keras API

# 지도 학습(supervised learning)
- KNeighborsClassifier
- numpy
- test set
- train Set
- Data preprocessing
- Scaling
- KNN
- Z점수
- R2 결졍계수
- 회귀 모델
- overfitting과 underfitting
- LinearRegression
- Panda
- 로지스틱 회귀
- 다중 분류
- LogisticRegression
- 시그모이드 함수
- Decision Tree
- 랜덤 포레스트 앙상블
- 그래디언트 부스팅
- 사이킷 런

# 비지도 학습(unsupervised learning)
- 군집(clustering)
- K-Means
- 주성분 분석

# 딥 러닝
- 심층 신경망
- 합성곱 신경망
- 케라스 API
